ci/setup_ci_environment.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+set -e
+apt-get update 
+apt-get install --no-install-recommends --no-install-suggests -y \
+                build-essential \
+                cmake \
+                pkg-config \
+                git \
+                ca-certificates \
+                curl \
+                automake \
```

Comment:
We can probably reduce this list. I don't think we need autotools.

Reply:
removed the extra packages